### DISCUSSION TITLE
Import old data from travelynx

### DISCUSSION THREAD

**Discussion by moritzan** [Traewelling/traewelling#2020] [Ideas / Feature Requests]
**Describe the solution you'd like**

Eine Möglichkeit, alle Daten von vorherigen Farhten von Travelynx auf Träwelling migrieren zu können.

**Comment by ErikUden** [comment]
Travelynx erlaubt es alle Daten zu exportieren, zudem hat es API Endpunkte um auf alle stattgefundenen Daten zuzugreifen! Das übertragen alter Fahrten und somit das Synchronisieren aller Daten von Travelynx auf Träwelling könnte Hand in Hand mit dem Manuellen hinzufügen von Fahrten vereinfacht werden. 

https://github.com/Traewelling/traewelling/issues/1429

As soon as adding Travels of the past has become possible, syncing rides from Travelynx to Träwelldroid will be a lot easier!
